rules/windows/process_creation/process_creation_msdeploy.yml

```diff
@@ -0,0 +1,34 @@
+title: Msdeploy.exe LOLBIN
+status: experimental
+id: d80d5c81-04ba-45b4-84e4-92eba40e0ad3
+author: Beyu Denis, oscd.community
+date: 2020/10/18
+description: launch binary via msdeploy.exe
+references:
+    - https://github.com/LOLBAS-Project/LOLBAS/blob/master/yml/OtherMSBinaries/Msdeploy.yml
+    - https://twitter.com/pabraeken/status/995837734379032576
+    - https://twitter.com/pabraeken/status/999090532839313408
+tags:
+    - attack.execution
+    - attack.t1218
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        ProcessCommandline|contains|all:
```

Comment:
ProcessCommandline should be replaced with CommandLine to maintain the standard of the process_creation category